### DISCUSSION TITLE
docker environment

### DISCUSSION THREAD

**Discussion by antoniotvr** [keycloakify/keycloakify#420] [General]
Hi.

We have a keycloak instance that is build using a docker file.

So my keycloak is running inside a container.

In this case how can I use keycloakify to customize theme and various messages of keycloak pages (for example the login disclaimer message)?

Thank you

**Comment by garronej** [comment]
Hello @antoniotvr,  

There's this video that explains it quite well: https://www.youtube.com/watch?v=x3ux2JM1Bxk
